### DISCUSSION TITLE
please tell me your favorite mahjong yaku

### DISCUSSION THREAD

**Discussion by shiro0309** [NAIST-SE/AttractingContributors#113] [General]
Come to my fork:
https://github.com/shiro0309/AttractingContributors-Mahjong

Add to my README file.

I look forward to your contributions

**Comment by KDR121** [comment]
I add AllSimples!
I'd be happy to help [here](https://github.com/KDR121/AttractingContributors), too.

**Comment by 9002taicher** [comment]
I added my favorite one to your file. Plz check my request.
And I'm grad you'll come here.
Thank you!
